modules/EvManager/main/car_simulation.cpp

```diff
@@ -234,21 +234,23 @@ bool CarSimulation::iso_stop_charging(const CmdArguments& arguments) {
 }
 
 bool CarSimulation::iso_wait_for_stop(const CmdArguments& arguments, size_t loop_interval_ms) {
-    if (!sim_data.sleep_ticks_left.has_value()) {
-        sim_data.sleep_ticks_left =
-            std::stoll(arguments[0]) * static_cast<long>(1 / static_cast<float>(loop_interval_ms)) + 1;
+    if (not sim_data.sleep_ticks_left.has_value()) {
```

Comment:
Not introduced in this PR, but std::stoll et al are used without catching exceptions here and in other places in EVManager. We should add try catch everywhere probably in a separate PR.

Reply:
Good point! I'll plan that.